jobrunner/lib/docker.py

```diff
@@ -54,13 +54,16 @@ def docker(docker_args, timeout=DEFAULT_TIMEOUT, **kwargs):
     except subprocess.TimeoutExpired as e:
         raise DockerTimeoutError from e
     except subprocess.CalledProcessError as e:
-        stderr = e.stderr
-        if isinstance(e.stderr, bytes):
-            stderr = stderr.decode("utf8")
+        output = e.stderr
+        if output is None:
+            output = e.stdout
+        if isinstance(output, bytes):
+            output = output.decode("utf8")
```

Comment:
If we want to be _super_ robust here we should probably use `.decode("utf8", "ignore")` so that malformed output doesn't cause another exception here.

Reply:
done